src/lib/Browser/Assert/ElementAssert.php

```diff
@@ -83,4 +83,22 @@ public function isVisible(): ElementInterface
 
         return $this->element;
     }
+
+    public function hasClass(string $expectedClass): ElementInterface
+    {
+        $actualClass = $this->element->getAttribute('class');
+
+        Assert::assertTrue($this->element->hasClass($expectedClass),
+            sprintf(
+                "Failed asserting that element with %s locator :'%s' '%s' has '%s' class, instead class is '%s'",
```

Comment:
```suggestion
                "Failed asserting that element with %s locator '%s': '%s' has '%s' class, instead class is '%s'",
```
To make it consistent with other messages:
https://github.com/ezsystems/BehatBundle/blob/7b85243ca9dd7edbe056f2b89f8202c19d12401a/src/lib/Browser/Assert/ElementAssert.php#L40
https://github.com/ezsystems/BehatBundle/blob/7b85243ca9dd7edbe056f2b89f8202c19d12401a/src/lib/Browser/Assert/ElementAssert.php#L60